src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -338,6 +345,9 @@ public DynamicJsonValue ToJson()
             }
         }
 
+        /// <summary>
+        /// Represents a delete operation in a time series, allowing data points within a specific range to be removed.
+        /// </summary>
         public sealed class DeleteOperation
         {
             public DateTime? From, To;
```

Comment:
need to write something here as well
does `From` & `To` included in the deletion?

Reply:
done